AmericanMedicalAssociation/ama-style-guide-2
# EWL-6447 Themes 50/50 video hub card

## JIRA Ticket(s)
- [EWL-6447: Bug: Hub page - 50/50 - theme](https://issues.ama-assn.org/browse/EWL-6447)


## Description:
Theme 50/50 video hub card

## To Test:
- switch your SG2 branch to `bugfix/EWL-6447-theme-50-50-video` (https://github.com/AmericanMedicalAssociation/ama-d8/pull/983)
- run `gulp serve`
- switch your D8 branch to `bugfix/EWL-6447-theme-50-50-video`
- enable local SG2 in your D8 instance
- run `drush @one.local cr`
- visit http://ama-one.local/node/24416
- edit the page
- add a 50/50 video hub card 
- save
- observe the 50/50 video hub card is now themed


## Automated Test:
n/a

## Style Guide:
The PR should look like this https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-hub-hero-as-fifty-fifty-video

## Relevant Screenshots/GIFs:
<img width="1139" alt="screen shot 2018-10-30 at 3 58 15 pm" src="https://user-images.githubusercontent.com/2271747/47750105-a2216680-dc5c-11e8-8831-3ff468a0b57d.png">


## Additional Notes:
Anything more to add?


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
